### DISCUSSION TITLE
How to apply transforms on Data object ?

### DISCUSSION THREAD

**Discussion by QuanticDisaster** [pyg-team/pytorch_geometric#3142] [Q&A] [answered]
Hi,

Just a question on the transforms objects. While there are examples for Datatset objects, I don't see in the doc how to apply them directly to dataloaders or data objects. At the end I used `.__call__(data)` of the transform element, but if it is the right way, I find it a bit obscure to guess, maybe an update to the documentation would be needed, maybe as indication in the class doc ? https://pytorch-geometric.readthedocs.io/en/latest/modules/transforms.html#torch_geometric.transforms.BaseTransform

**Comment by rusty1s** [comment, marked as answer]
That is a great idea. I added more information to the documentation, see [here](https://pytorch-geometric.readthedocs.io/en/latest/modules/transforms.html#torch_geometric.transforms.BaseTransform). Let me know if this resolves your issues :)

**Reply by QuanticDisaster** [reply]
It works well, thx !
